src/what.sh

```diff
@@ -38,7 +43,8 @@ function _What_alias { (
     if [[ $print_definition == true ]]; then
         # Print the *current* definition.
         _What_indent 2
-        printf "definition: "
+    # shellcheck disable=SC2059  
```

Comment:
No reason to do this when you can use `%b` instead: `printf "%bdefinition%b: " "$GREEN" "$NORMAL"`